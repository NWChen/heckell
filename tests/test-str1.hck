let s : string;
s = "hello world!";

"hello world\nThis is a new line";

"\"special\" \t 'test\'";

"\72\101\108\108\111";
