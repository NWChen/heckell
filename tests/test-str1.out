let s: string
s = "hello world!"
"hello world
This is a new line"
""special" 	 'test'"
"Hello"
